backend/src/aggregator.rs

```diff
@@ -20,9 +21,27 @@ pub struct ChainEntry {
     addr: Addr<Chain>,
     label: Label,
     network_id: Option<Label>,
+    /// Node count
     nodes: usize,
+    /// Maximum allowed nodes
+    max_nodes: usize,
 }
 
+lazy_static! {
+    /// Labels of chains we consider "first party". These chains are allowed any
+    /// number of nodes to connect.
+    static ref FIRST_PARTY_NETWORKS: HashSet<&'static str> = {
+        let mut set = HashSet::new();
+        set.insert("Polkadot");
+        set.insert("Kusama");
+        set.insert("Westend");
+        set.insert("Rococo");
```

Comment:
This needs some vetting. Is that the full list of networks we want to consider "first party"?

Reply:
Looks fine to me, at least; we also run Canvas but I think we can ignore that for now. The only one I'm not super sure about is "Local Testnet" but that's well under 500 at the moment so...